### ISSUE TITLE
How to exclude dates when repeating an event?

### ISSUE THREAD

**Issue by devboomtech** [adamgibbons/ics#219]
How to exclude dates when repeating an event?

**Comment by jacobwise** [comment]
I got this working by appending the `EXDATE` to the `RRULE`

https://icalendar.org/iCalendar-RFC-5545/3-8-5-1-exception-date-times.html

Make sure you add a line break as it's a separate attribute to the event

```
...
RRULE:FREQ=DAILY;INTERVAL=1;UNTIL=20221126T000000Z
EXDATE:20221123T210000
END:VEVENT
END:VCALENDAR

```

**Comment by adamgibbons** [comment]
Resolved by #246
